app/src/test/java/com/onboarding/parkingsystemkotlin/mvp/presenter/ParkingPresenterTest.kt

```diff
@@ -37,7 +37,15 @@ class ParkingPresenterTest {
         verify(view).showNewReservationActivity()
```

Comment:
on this test you have to add the validation for the removeOldReservations too

Reply:
done